Marlin/src/module/planner.cpp

```diff
@@ -2008,8 +2008,8 @@ bool Planner::_populate_block(
   #endif
 
   SECONDARY_AXIS_CODE(
-    dm.i = (dist.i > 0), dm.j = (dist.j > 0), dm.k = (dist.k > 0),
-    dm.u = (dist.u > 0), dm.v = (dist.v > 0), dm.w = (dist.w > 0)
+    dm.i = (dist._i > 0), dm.j = (dist._j > 0), dm.k = (dist._k > 0),
+    dm.u = (dist._u > 0), dm.v = (dist._v > 0), dm.w = (dist._w > 0)
```

Comment:
The `AxisBits` struct members with underscores are just placeholders to fill out the `union` and are not intended to be acknowledged or used in code.